<!doctype html>
 <html > 
  <meta charset="utf-8" />
  <!--[if IE]>
  <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
  <![endif]-->
  <title>HLPID</title>
  <meta name="description" content="" />
  <meta name="author" content="" />
  <meta name="viewport" content="width=device-width" />
   <link href="img/log_title.jpg" rel="SHORTCUT ICON">
  
  <link rel="stylesheet" href="css/bootstrap.css" />
  <link rel="stylesheet" href="dist/bootstrap-table.css" />
  <link rel="stylesheet" href="css/jquery-confirm.min.css" type="text/css" media="all">
  <link rel="stylesheet" href="style.css" />
  
  <script src="includes/js/libs/jquery-3.1.1.js"></script>
  <script type="text/javascript" src="dist/jquery-confirm.min.js"></script>
  <script src="includes/js/jquery.validate.js" type="text/javascript"></script>
  
  
  <script src="includes/js/libs/bootstrap/bootstrap.js"></script>
  <script src="dist/bootstrap-table.js"></script>
  
  
   <link rel="stylesheet" href="style.css" />
   
   
   
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" /></head>
<body>
   <div id="header"></div>
   
   <!-- #nav -->
   <!-- To prevent the content from "disappearing" under the nav when it is viewed at the top -->
   <div class="clear after_nav"></div>
   <!-- Start Content -->
   <div class="container margin-top mobile-no-margin-top" >
	       
	       <div class="col-md-10" style="text-align: justify;font-size:20px;">
      

	
<h3>General introduction</h3>
	<h5>What are the versions of Ensembl gene annotation and UniProt proteins used in this database?</h5>
		<p>
		
           Currently, Ensembl gene annotation version 85 and 94,817 lncRNAs were used in HLPID. The 156,773 human proteins of UniProt (including Swiss-Prot and TrEMBL) database were downloaded on Feb. 15th, 2017.
		
		</p>
	
	
	<h5>What are the versions of dbSNP and Cosmic utilized in HLPID?</h5>
		  <p>
		  
		  The versions of dbSNP and Cosmic used in HLPID are version-147 and version-79, respectively. 
		  
		  </p>
	
	<h5>How did you calculate the interactions between lncRNAs and proteins?</h5>
		<p>
		
			We employed RNAcommender (April 2nd, 2017 ) to calculate the genome-wide interactions between lncRNAs and proteins. Only those interactions with score higher than 0.02 were included in HLPID. RNAcommender is an efficient software for predicting lncRNA-protein interactions. More details about RNAcommender software can be found in https://github.com/gianlucacorrado/RNAcommender.
		
		</p>
	
	<h5>How did you annotate the SNPs and mutations to those lncRNAs?</h5>
		<p>
		
          First, the SNPs in dbSNPs database and the mutations in Cosmic database were downloaded. Then we annotated those SNPs and mutations based on the Ensembl gene annotation version 85 using ANNOVAR (released on Feb. 1st, 2016). Only those SNPs and mutations mapped to the promoter regions (1kb of transcription start site of genes), exonic and intronic regions of lncRNAs were considered.
    
    </p>
		
	<h5>How did you conduct the survival analysis using the TCGA data?</h5>
	
	<p>
	
		Gene expression and clinical data were downloaded from GDC (Genomic Data Commons) database.  <br/>
		In "Single-gene" mode, for a given cancer, we classified all patients into two different groups according to the expression of each gene. Based on the survival time difference between the two classes, we could determine whether the expression level of one gene can affect the patientsâ€™ survival time significantly.
		In "Multi-genes" mode, for a given cancer, we established risk score formula by performing multivariable Cox regression analysis using the genes selected from the analyses of differential expression calling and univariate Cox regression. Then, we divided patients into high-risk and low-risk groups using the median risk score as cutoff.
		
    
    </p>
		
	<h5>How did you predict the associations between lncRNAs and diseases?  </h5>
	<p>
		
		The associations between lncRNAs and diseases in HLPID databases were predicted using LncDisease (version 1.41).   
    
    
    
    </p>	
		
   <h5>How did you identify the deleterious noncoding variants located in corresponding regions of lncRNAs?  </h5>
	<p>
	
		We employed LINSIGHT (April 2nd, 2017) to assess the deleteriousness of those noncoding variants located in the relevant regionso of lncRNAs.
   
    </p>	
	
<h5>How often will you update HLPID?  </h5>
	<p>
	
		We employed LINSIGHT (April 2nd, 2017) to assess the deleteriousness of those noncoding variants located in the relevant regionso of lncRNAs.
   
    </p>	
	
		
</div>

</div>


      
   <br/>
    <br/>
     <br/>
     <br/>
  <div class="clearfix"></div>
   <div class="clearfix"></div>
   <div id="footer"></div>
  
  
   
<script type="text/javascript">




$(function()
{

	$("#header").load("pages/header.html");
	$("#footer").load("pages/footer.html");
	
})







</script>
   
   
   


</body>
</html>